src/utils/transportation-color.ts

```diff
@@ -17,38 +18,38 @@ export default function transportionColor(
       const code = Number(publicCode);
       if (code < 100)
```

Comment:
Btw. There is a transportation submode on service journey which I belive can say if it is a regional bus or not.

Reply:
Ah - Will try and figure out if there's any good reason to why the code check was used instead of that initially  :monocle_face:

Reply:
I believe we didn't know better. 😬  cc @cbrevik

Reply:
As far as I know, it actually didn't correctly specify correctly if it was a regional bus or not. If it does now, great 🎉. I haven't checked in a while.

Reply:
To be fair: I haven't actually checked the data. Let's just make another issue for this

Reply:
How would one know for sure? Do you know of some specific lines that would appear as local when regional, or the other way around?

Reply:
Don't remember any specific lines no, just checked trips like Trondheim - Stjørdal. A quick search now on the same trip seems like it has the correct `transportSubmode` under service-journey, so seems like they've fixed it 🎉 Earlier all of them said `localBus`.

```
		"serviceJourney": {
				"id": "ATB:ServiceJourney:310_200610136946289_30_200127114109271",
				"journeyPattern": {
					"line": {
						"id": "ATB:Line:200610136922955_310_200127114109271",
						"name": "Stjørdal - Trondheim - Orkanger",
						"notices": [],
						"publicCode": "310",
						"transportMode": "bus",
						"transportSubmode": "regionalBus"
					},
					"notices": []
				},
				"notices": [],
				"publicCode": "",
				"privateCode": "",
				"transportSubmode": "regionalBus"
			},
			"situations": []
		},
```

Reply:
Nice :+1:

Reply:
I've changed so transportSubmode is now used instead of publicCode. Updated all usages as well.